xunit-autolabeler-v2/ast_parser/python/source_parsers/__init__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2020 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import sys
+sys.path.append('..')
```

Comment:
I'll add a `README`

Reply:
Comment added.

Reply:
Can we set PYTHONPATH when running the test in the test driver instead of always modifying `sys.path`?

Reply:
So if we make the _tests_ the special case instead of `python_bootstrap.py`, we'd have to use a lower **(and less-standardizable)** root directory (e.g. `import lib.constants`) instead of a consistent package-level root (e.g. `import ast_parser.lib.constants`.)

Personally, I'm a fan of consistency - but let me know if you're not. If that's the case, I can look into configuring this via `conftest.py`. We might also be able to use [this pypi package](https://pypi.org/project/pytest-pythonpath/) alongside `pytest.ini`, but it doesn't look very well-maintained.

Reply:
I don't see `python_bootstrap.py` in this PR. Can you add it?

Reply:
I checked out your branch, and comment out the `sys.path` hack, then did this:

```sh
$ cd xunit-autolabeler-v2
$ pytest
```

The tests fails with this error message:
```
E       AssertionError: assert 48 == 47
ast_parser/python/source_parsers/flask_router_test.py:76: AssertionError
```

Although the test failed, I could run the test.

My question is: Do we really need this `sys.path` hack?